command.go

```diff
@@ -121,7 +122,7 @@ func (c *cancellableCommand) Stop(gracePeriod time.Duration) {
 }
 
 // NewCommand returns a new command based on passed CommandInfo.
-func NewCommand(commandInfo mesos.CommandInfo, env []string, options ...func(*exec.Cmd) error) (Command, error) {
+func NewCommand(commandInfo mesos.CommandInfo, env hook.Env, options ...func(*exec.Cmd) error) (Command, error) {
```

Comment:
`Env` should not be defined in `hook` package

Reply:
Where do you suggest to put it? executor.go? command.go?

Reply:
Reverted back to []string